packages/flutter_test/lib/src/binding.dart

```diff
@@ -106,6 +111,32 @@ mixin TestDefaultBinaryMessengerBinding on BindingBase, ServicesBinding {
   }
 }
 
+/// An accessibility announcement made by the Flutter framework.
+///
+/// This class is mostly used by the testing api to store the announcements
+/// in a user-friendly structure so that the code consumers can check the details
+/// of announcements in their tests.
+///
+/// See [TestWidgetsFlutterBinding.takeAnnouncements].
```

Comment:
I would primarily point to `WidgetTester. takeAnnouncements` and `WidgetTester.peekAnnouncement`, since it's more likely to be used by test code.

Reply:
Done! Now it's pointing only to `WidgetTester. takeAnnouncements` since I've removed `WidgetTester.peekAnnouncement`.